### DISCUSSION TITLE
When was the world's first webcam created?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#721] [Galaxy Trivia] [answered]
**Bonus:** What was it set up to do?

**Comment by peterbaccaro** [comment, marked as answer]
The world's first webcam created in **1993** and it was setup to **allow people to see when the coffee pot was full or empty**.

A group of scientists at the University of Cambridge were credited with inventing the first web cam in 1993 - and it was all because of a coffee pot!

Many of the scientists worked on different floors in the same building, and when they wanted to get a cup of coffee they had to travel quite far to the main computer room to get a drink.

Sometimes when they would get there the coffee pot would be empty, and this would be very disappointing.

So they decided to set up a camera which would allow people to see when the coffee pot was full or empty to save them making a wasted journey.

On 22 November 1993 the camera made it onto the World Wide Web.

Dr Martyn Johnson couldn't access the coffee camera, so he built some computer code which meant he could watch the video online instead!

And just like that the first webcam was created.

![coffee](https://github.com/ParanoidUser/yolo/assets/74666079/6602ca9b-a72c-46d7-a680-70f08a85c98d)

**Reply by ParanoidUser** [reply]
Fabulous! 👏👏

**Comment by SANKARAMDAS** [comment]
The coffee pot camera

A group of scientists at the University of Cambridge were credited with inventing the first webcam in 1993 - and it was all because of a coffee pot!
Dr. Quentin Stafford-Fraser and Paul Jardetsky set up a camera to take 129x129 sized pictures of the coffee pot three times a minute, to help late-night studiers and programmers keep an eye on coffee levels.

**Comment by thuongtruong1009** [comment]
The world's first webcam was created in 1991.

The world's first webcam was set up to monitor a coffee pot in the Computer Science Department at the University of Cambridge. The purpose was to provide a live video feed of the coffee pot, allowing individuals to check if there was coffee available before making the trip to the break room. This way, they could save time by avoiding unnecessary trips when the coffee pot was empty.
